mis_bot/misbot/general.py

```diff
@@ -178,8 +178,10 @@ def cancel(bot, update):
 def unknown(bot, update):
     """Respond to incomprehensible messages/commands with some canned responses."""
     can = ["Seems like I'm not programmed to understand this yet.", "I'm not a fully functional A.I. ya know?",
-    "The creator didn't prepare me for this.", "I'm not sentient...yet! 🤖", "Damn you're dumb.", "42",
-    "We cannot afford machine learning to make this bot smart!", "We don't use NLP.", "I really wish we had a neural network."]
+        "The creator didn't prepare me for this.", "I'm not sentient...yet! 🤖", "Damn you're dumb.", "42",
+        "We cannot afford machine learning to make this bot smart!", "We don't use NLP.", "I really wish we had a neural network.",
+        "You make me wish I had middle fingers.", "Sorry, did you say something? I wasn't listening.",
+        "Are you always such an idiot, or do you just like to show off here?", "Talk to the hand. ✋"]
```

Comment:
```suggestion
        "Are you always such an idiot, or do you just like to show off here?", "Talk to the hand. ✋",]
```

The trailing comma will make sure when in the future someone updates the list, the git diff will only contain additions. [Read more about it here](https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8).